sections/changes.include

```diff
@@ -13,6 +13,9 @@
 <h3 id="changes-wd4">Changes since the <a href="https://www.w3.org/TR/2018/WD-html53-20180426/">HTML 5.3 Third Working Draft</a></h3>
 <dl>
   <dt><a href="https://github.com/w3c/html/pull/1479"><code>http-equiv="set-cookie"</code> has no effect</a>.</dt>
+  <dt><a href="https://github.com/w3c/html/pull/1489">Define <code>HTMLOrSVGElement</code> mixin interface</a>.</dt>
+  <dd>Fixed <a href="https://github.com/w3c/html/issues/1299">issue 1299</a>
```

Comment:
Should this mention moving the nonce handling? If so, maybe tweak that - merging now anyway...